includes/ocos.h

```diff
@@ -6,6 +6,7 @@
 #define ORT_API_MANUAL_INIT
 #define EXCLUDE_REFERENCE_TO_ORT_DLL
 #include "onnxruntime_cxx_api.h"
+#include "onnxruntime_cxx_inline.h"
```

Comment:
the *_inline.h is included at the footer of *_cxx_api.h, this change is unnecessary.

Reply:
done